### ISSUE TITLE
AttributeError("'Server' object has no attribute '_triggers'")

### ISSUE THREAD

**Issue by nodtem66** [Kitware/trame-tutorial#4]
Hello, 
I tried to run 03_html/solution_buttons.py. Everything worked except the reset camera button. After I clicked it, I got the exception from the browser:

```json
{
    "code": -32001,
    "message": "Exception raised",
    "data": {
        "method": "trame.trigger",
        "exception": "AttributeError(\"'Server' object has no attribute '_triggers'\")",
        "trace": "Traceback (most recent call last):\n  File \"C:\\Users\\XXX\\miniconda3\\envs\\trame\\lib\\site-packages\\wslink\\protocol.py\", line 334, in onMessage\n    results = await results\n  File \"C:\\Users\\XXX\\miniconda3\\envs\\trame\\lib\\site-packages\\trame_server\\protocol.py\", line 185, in trigger\n    if name in self.server._triggers:\nAttributeError: 'Server' object has no attribute '_triggers'\n"
    }
}
```

and from the cmd line:
```
Exception raised
AttributeError("'Server' object has no attribute '_triggers'")
Traceback (most recent call last):
  File "C:\Users\XXX\miniconda3\envs\trame\lib\site-packages\wslink\protocol.py", line 334, in onMessage
    results = await results
  File "C:\Users\XXX\miniconda3\envs\trame\lib\site-packages\trame_server\protocol.py", line 185, in trigger
    if name in self.server._triggers:
AttributeError: 'Server' object has no attribute '_triggers'
```

This is the environment I used to run this example:
* Window10 22H2
* Chrome Version 119.0.6045.200 (Official Build) (64-bit)
* trame-tutorial commit/bcd6c1372b48a728fbe87c07da10b4d924e110ab
* miniconda3 23.10.0
* conda list:
```
aiohttp                   3.9.1                    pypi_0    pypi
aiosignal                 1.3.1                    pypi_0    pypi
async-timeout             4.0.3                    pypi_0    pypi
attrs                     23.1.0                   pypi_0    pypi
bzip2                     1.0.8                hcfcfb64_5    conda-forge
ca-certificates           2023.11.17           h56e8100_0    conda-forge
contourpy                 1.2.0                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
fonttools                 4.46.0                   pypi_0    pypi
frozenlist                1.4.0                    pypi_0    pypi
idna                      3.6                      pypi_0    pypi
importlib-resources       6.1.1                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
libffi                    3.4.2                h8ffe710_5    conda-forge
libsqlite                 3.44.2               hcfcfb64_0    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
matplotlib                3.8.2                    pypi_0    pypi
more-itertools            10.1.0                   pypi_0    pypi
multidict                 6.0.4                    pypi_0    pypi
numpy                     1.26.2                   pypi_0    pypi
openssl                   3.2.0                hcfcfb64_1    conda-forge
packaging                 23.2                     pypi_0    pypi
pillow                    10.1.0                   pypi_0    pypi
pip                       23.3.1             pyhd8ed1ab_0    conda-forge
pyparsing                 3.1.1                    pypi_0    pypi
python                    3.9.18          h4de0772_0_cpython    conda-forge
python-dateutil           2.8.2                    pypi_0    pypi
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
six                       1.16.0                   pypi_0    pypi
tk                        8.6.13               h5226925_1    conda-forge
trame                     3.3.0                    pypi_0    pypi
trame-client              2.13.0                   pypi_0    pypi
trame-server              2.13.0                   pypi_0    pypi
trame-vtk                 2.6.2                    pypi_0    pypi
trame-vuetify             2.3.1                    pypi_0    pypi
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
vc                        14.3                h64f974e_17    conda-forge
vc14_runtime              14.36.32532         hdcecf7f_17    conda-forge
vs2015_runtime            14.36.32532         h05e6639_17    conda-forge
vtk                       9.3.0                    pypi_0    pypi
wheel                     0.42.0             pyhd8ed1ab_0    conda-forge
wslink                    1.12.4                   pypi_0    pypi
xz                        5.2.6                h8d14728_0    conda-forge
yarl                      1.9.4                    pypi_0    pypi
zipp                      3.17.0                   pypi_0    pypi
```

**Comment by jourdain** [comment]
Oops, you just caught us in the middle of an update of a cross dependency between trame-server and trame-client.

For now, can you fix some versions while we are working on fixing it (@alesgenova)

`pip install "trame-server<2.13" "trame-client==2.13"`

**Comment by jourdain** [comment]
Actually that was a good catch. It is now fixed in trame-server>=2.13.1`...

Thanks for reporting the issue.
